docs/changelog.rst

```diff
@@ -11,6 +11,15 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+unreleased
```

Comment:
We don't write the changelog directly, but instead use towncry... See the PR template for more details.

Reply:
Removed the change from the changelog.